### PR TITLE
[ST-839] set test framework  name to header

### PR DESCRIPTION
**Pull request by Konboi**
set test-runner name when cli requests to the API.

### CODE REVIEW

File: launchable/utils/http_client.py [hunk 1]
```diff
@@ -34,7 +35,12 @@ def request(self, method, path, **kwargs):
...
     def _headers(self):
-        return {
+        h = {
             "User-Agent": "Launchable/{} (Python {}, {})".format(__version__, platform.python_version(), platform.platform()),
-            'Authorization': 'Bearer {}'.format(self.token)
+            'Authorization': 'Bearer {}'.format(self.token),
         }
+
+        if self.test_runner != "":
+            h["Launchable-Test-Runner"] = self.test_runner
```
Comment (Konboi) [inline launchable/utils/http_client.py:44]: What do you think this field name?

Comment (yoshiori) [reply launchable/utils/http_client.py:44]: I recommend you to use `launchable-test-runner`.
I want to use a name that can also be used for http/2 for future reference.

https://tools.ietf.org/html/rfc7540#section-8.1.2
>  A request or response containing uppercase  header field names MUST be treated as malformed

Comment (Konboi) [reply launchable/utils/http_client.py:44]: OK, I thought that I should arrange the name with other headers. but I'll use http2 rule

File: launchable/commands/subset.py [hunk 2]
```diff
@@ -198,7 +198,8 @@ def run(self):
                     path = "/intake/organizations/{}/workspaces/{}/subset".format(
                         org, workspace)
 
-                    client = LaunchableClient(token)
+                    client = LaunchableClient(
+                        token, test_runner=context.invoked_subcommand)
```
Comment (Konboi) [inline launchable/commands/subset.py:202]: Is there any other way to get a test-runner name?

**Review by ninjinkun — APPROVED** [review]
Looks good. I prefer to add a test to assert the header on this or another PR

**Comment by Konboi** [comment]
Thank you for your review! I added a unit test and it passed so, I'll merge and ship it

**Pull request merged**